cmd/mimir/help-all.txt.tmpl

```diff
@@ -251,9 +251,9 @@ Usage of ./cmd/mimir/mimir:
     	Maximum size of the Alertmanager configuration for a tenant. 0 = no limit.
   -alertmanager.max-dispatcher-aggregation-groups int
     	Maximum number of aggregation groups in Alertmanager's dispatcher that a tenant can have. Each active aggregation group uses single goroutine. When the limit is reached, dispatcher will not dispatch alerts that belong to additional aggregation groups, but existing groups will keep working properly. 0 = no limit.
-  -alertmanager.max-grafana-config-size-bytes int
+  -alertmanager.max-grafana-config-size-bytes value
```

Comment:
Is "value" clear for end-users?

Reply:
good point, added a case to handle `flagext.Bytes` now. It does change the descriptor of all flags using it though

Reply:
Dropped the commit as discussed in slack. I'll create another PR and propose an appropriate format for that 👍